packages/themes/src/themes/shadcn.ts

```diff
@@ -0,0 +1,34 @@
+import { experimental_createTheme } from '../createTheme';
+
+export const shadcn = experimental_createTheme({
+  cssLayerName: 'components',
```

Comment:
Because we know we are using TW via shadcn, we can simply mark the CSS layer name as `components` to not need to specify the `cssLayerName` since we are using `elements` within the theme.